lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -63,7 +65,7 @@
     description:
       - The port number on which each of the cache nodes will accept connections
     required: false
-    default: None
+    default: 11211
```

Comment:
it failed with none because it wanted an int, so I chose a port that will likely work with other defaults (memcache)